clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -71,6 +72,23 @@ def reload_state(self) -> None:
         self._nodes_by_ip = self._get_nodes_by_ip()
         self._pods_by_ip = self._get_pods_by_ip()
 
+    def get_removed_nodes_since_last_reload(self) -> Set[KubernetesNode]:
+        try:
```

Comment:
Can you rebase on master?  We now store a copy of the Kubernetes client object, so we shouldn't need to do this `try/except` block anymore.